meetings/2024-08-13.md

```diff
@@ -0,0 +1,24 @@
+# TC Meeting 2024-08-06
+
+Present: Cornelius, Adrian, Max, Loic, Stephanie, Tristram; Excused: Florian, Frederik
+
+## Agenda
+
+* Chair of Technical Committee
+* Onboarding
+  * Tracking: https://github.com/OpenRailAssociation/technical-committee/labels/Onboarding
+* DCO documentation (https://github.com/OpenRailAssociation/technical-committee/pull/105)
+* GitHub accounts (https://github.com/OpenRailAssociation/technical-committee/pull/115)  
+
+## Protocol
+
+* Chair of the Technical Committee
+  * Chairmainship of the Technical Committee passes from Cornelius to Loic for the second half of the election period as decided at the beginning of the year
+  * Loic will moderate the meeting starting next week
+  * Cornelius will continue to prepare agenda and protocol for now
+* Onboarding
+  * Tracking: https://github.com/OpenRailAssociation/technical-committee/labels/Onboarding
+  * We add one contact person from the project and one from the TC for each issue
+  * TODO: Send Max the GitHub handle of the contact person of the project, so we can add them to the organisation
```

Comment:
This is already done in https://github.com/OpenRailAssociation/openrail-org-config/pull/13

```suggestion
```